packages/components/tests/dummy/app/templates/components/pagination.hbs

```diff
@@ -23,94 +18,316 @@
 <section>
   <h3 class="dummy-h3" id="component-api"><a href="#component-api" class="dummy-link-section">§</a> Component API</h3>
   <p class="dummy-paragraph">
-    The
-    <code class="dummy-code">Pagination</code>
-    component is a wrapper component for the Pagination-related sub-components, each with their own APIs:
-  </p>
+    The term «<code class="dummy-code">Pagination</code>
+    component» is an umbrella term used to indicate a set of Pagination-related components and sub-components, each with
+    their own APIs.</p>
```

Comment:
I would use regular English quotes: 
“”
vs.
«»
which I think of as being "French quotes" (may be used in other languages too but I took French in high school & college).